hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/tasks/FileSizeCountTask.java

```diff
@@ -162,10 +162,10 @@ private void updateCountFromDB() {
           updateUpperBoundCount(omKeyInfo, DELETE);
           break;
 
-        default: LOG.trace("Skipping DB update event : " + omdbUpdateEvent
-                  .getAction());
+        default: LOG.trace("Skipping DB update event : {}",
+            omdbUpdateEvent.getAction());
         }
-      } catch (IOException e) {
+      } catch (Exception e) {
```

Comment:
Why Exception is changed here, any reason?

Reply:
The exception type was wrong. IOException was not being thrown. And in this method, we have to capture all Exceptions.